Mathlib/RingTheory/Discriminant.lean

```diff
@@ -63,6 +63,7 @@ section Discr
 `discr A ι b` as the determinant of `traceMatrix A ι b`. -/
 -- Porting note: using `[DecidableEq ι]` instead of `by classical...` did not work in
 -- mathlib3.
+@[stacks 0BIM]
```

Comment:
I don't think this tag is right.

Reply:
Thanks! This is the discriminant of an element with respect to a certain basis. The discriminant of a field should its value when a = 1. Or more precisely, its image in `K/(K*)^2`.

Reply:
Removed.